low-level/files/opt/seagate/sspl/setup/resource_map/lr2/server.py

```diff
@@ -45,12 +50,16 @@ def __init__(self):
         self.sysfs.initialize()
         self.sysfs_base_path = self.sysfs.get_sysfs_base_path()
         self.cpu_path = self.sysfs_base_path + CPU_PATH
-        self.server_frus = {
+        hw_resources = {
```

Comment:
lets keep server_map, as this includes both hw, sw, pltform sensors 
same we had decided in https://github.com/Seagate/cortx-monitor/pull/423/files#diff-510a415283f8f42bff6012c3899cdf94bda9930a25d6ab587d1791ba7d8efa99R49